### ISSUE TITLE
Feature idea: in-game recipe book

### ISSUE THREAD

**Issue by RWsYiffyMind** [naelstrof/KoboldKare#170]
Right now, there's all these recipes that can be made in-game, but people either have to memorize them, or read some external steam forum post, or manually keep their own .txt file outside the game....   I think it would be both fun and convenient if there were some in-game recipe book, ideally one that gradually fills out with just the recipes you've tried on that save file.   It feels like this would encourage experimentation rather than people just looking up the list online to avoid how tedious manual tracking can get.

It could be as simple as being purely iconographic, with just little 🍆+🍈+❤️=📈 symbols, like the grimoire in vampire survivors.

I'm undecided if this would work better as something that pops up over the screen as UI, or if maybe it's even just a poster that would hang on the wall of the kitchen?

Not sure if I'd get around to this soon, but I thought I'd at least see what people think of the possibility.

**Comment by Godeken** [comment]
Agreed, but... You must discover the recipe before it shows in the book/poster. Gives it a challenge and then you can try add a challenge to figure out all the recipes.

**Comment by RWsYiffyMind** [comment]
Yeah sorry, that was what I meant by "one that gradually fills out with just the recipes you've tried on that save file".
